docker/haproxy/haproxy.cfg

```diff
@@ -40,10 +40,14 @@ frontend test_frontend
     http-request silent-drop if { var(txn.coraza.action) -m str drop }
     http-response silent-drop if { var(txn.coraza.action) -m str drop }
 
-    # Deny in case of an error, when processing with the Coraza SPOA
+    # Deny in case of an error, when processing with the Coraza SPOE
```

Comment:
The SPOE is the Haproxy part which talks to the SPOA. So I think this as correct.

Reply:
HAProxy talks about [SPOE/SPOP](https://github.com/haproxy/haproxy/blob/49ddd87d41b62797d50d7c59e843d0c0b0236eb5/doc/SPOE.txt#L300-L337) only.

There are `txn.coraza.err_code` and `txn.coraza.err_msg` for SPOA errors now.